judo-psm-generator-jackson-impl/src/main/resources/rest/impl/rest.service.impl.java.hbs

```diff
@@ -0,0 +1,80 @@
+{{> fragment.header.hbs }}
+
+package {{ namedElementImplRestPackageName self }};
+
+@javax.ws.rs.Path("/")
+public class {{ implClassName self }} implements {{ namedElementApiRestFqName self }} {
+
+
+    java.util.Optional<hu.blackbelt.judo.dispatcher.api.Dispatcher> __dispatcher = java.util.Optional.empty();
+
+    @Override
+    public void __bindDispatcher(hu.blackbelt.judo.dispatcher.api.Dispatcher dispatcher) {
+        this.__dispatcher = java.util.Optional.of(dispatcher);
+    }
+
+    @Override
+    public void __unbindDispatcher(hu.blackbelt.judo.dispatcher.api.Dispatcher dispatcher) {
+        this.__dispatcher = java.util.Optional.empty();
+    }
+
+{{# each self.operations as | operation | }}
+
+    {{# if operation.isStatelessAndHasNoInputParameter }}
```

Comment:
`@javax.ws.rs.Produces("application/json")` can go outside of the `if` block.

Reply:
[00ec87c](https://github.com/BlackBeltTechnology/judo-psm-generator-jackson/pull/1/commits/00ec87cbc991117419a9c0480c7bbf20867b0620)